packages/wallets/src/features/cashier/modules/Transfer/types/types.ts

```diff
@@ -0,0 +1,10 @@
+import { useWalletTransfer } from '../hooks';
+
+type TAccount = ReturnType<typeof useWalletTransfer>['activeWallet'];
```

Comment:
Pls move this to generic types folder

Reply:
checked with @thisyahlen-deriv
decided to leave this type here because:
1. this type is using custom `useWalletTransfer` which is not in api hooks
2. in future it can cause difficulties in separating `cashier` as a standalone repo